### PR TITLE
DAI 1.8.7.1 tutorial updates for Automatic Machine Learning Intro w/ DAI

### PR DESCRIPTION
**Pull request by Castro38**
**Updated the following using  Lab 9:  DAI 1.8.7.1** 
- Images (blurred out DAI version)
- Content to match images
- Updated settings (Accuracy, Time, Interpretability) to match DAI documentation wording
- Added hyperlinks to a selected group of settings 
- Tested all links, updated broken links and deleted links that were no longer available
- Updated any tutorial references to point to LMS tutorials

### CODE REVIEW

File: Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 1]
```diff
@@ -470,21 +472,22 @@ Note: To disable **assistant**, click on assistant again.
...
+9. [**Fold column**](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#fold-column) - The fold column is used to create the training and validation datasets so that all rows with the same Fold value will be in the same dataset. This column will be used if no validation dataset is provided and the model is trained using cross validation. 
+10. [**Weight column**](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#weight-column) - Column that indicates the observation/isntance weight (a.k.a. sample or row weight), if applicable.  
+11. [**Time Column**](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#time-column)(OFF by default) - Provides a time order(timestamps for observations). Used when data has a high dependency on time (such as seasonality or trend), and you want to treat this problem as a time series problem. 
 
 Continuing with our experiment:
 
+Type the name for your experiment `Titanic Classification Tutorial`
```
Comment (5675sp) [inline Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:489]: change to: 

Name your experiment as follows: `Titanic Classification Tutorial`

File: Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 2]
```diff
@@ -683,21 +688,21 @@ The ROC curve below shows Receiver-Operator Characteristics curve stats on valid
...
 ![experiment-results-roc-graph](assets/experiment-results-roc-graph.jpg)
 
-This ROC gives an Area Under the Curve or AUC of .8530. The AUC tells us that the model is able to separate the survivor class with an accuracy of 85.30%.
+This ROC gives an Area Under the Curve or AUC of .8472. The AUC tells us that the model is able to separate the survivor class with an accuracy of 84.72%.
 
-Learn more about the ROC Curve on [Machine Learning Experiment Scoring and Analysis Tutorial - Financial Focus: ROC](https://h2oai.github.io/tutorials/machine-learning-experiment-scoring-and-analysis-tutorial-financial-focus/#7).
+Learn more about the ROC Curve on [Machine Learning Experiment Scoring and Analysis Tutorial - Financial Focus: ROC](https://training.h2o.ai/products/tutorial-1b-machine-learning-experiment-scoring-and-analysis-tutorial-financial-focus).
 
-3\. Prec-Recall - Precision-Recall graph
+3\. Prec-Recall: Precision-Recall graph
```
Comment (5675sp) [inline Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:695]: Prec-Recall: Precision-Recall Graph

File: Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 3]
```diff
@@ -510,14 +513,17 @@ The survived attribute was selected because, as an insurance company, we want to
...
+
+    - [**Time**](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#time) - This specifies the relative time for completing the experiment (i.e., higher settings take longer). Early stopping will take place if the experiment doesn’t improve the score for the specified amount of iterations.
+
+    - [**Interpretability**](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#interpretability) -  TSpecify the relative interpretability for this experiment. Higher values favor more interpretable models. Changing the interpretability level affects the feature pre-pruning strategy, monotonicity constraints, and the feature engineering search space.
```
Comment (5675sp) [inline Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:520]: typo:

from: "TSpecify" change to: "Specify"

File: Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 4]
```diff
@@ -737,7 +742,7 @@ Hover over a point in the chart to view the quantile percentage and Kolmogorov-S
 
 For the K-S chart above, if we look at the top 60% of the data, the at-chance model (the dotted diagonal line) tells us that only 60% of the data was successfully separate between positives and negatives (survived and did not survived). However, with the model, it was able to do .499, or about 50% of the cases were successfully separated between positives and negatives.
```
Comment (5675sp) [inline Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:743]: where did you get .499?

File: Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md [hunk 5]
```diff
@@ -32,9 +32,9 @@ In this overview of Driverless AI, you will learn how to load data, explore data
...
+  - Need a **Two-Hour Test Drive** session? Follow the instructions [on this quick tutorial](https://training.h2o.ai/products/tutorial-0-getting-started-with-driverless-ai-test-drive) to get a Test Drive session started. After the Driverless AI Test Drive session starts, continue reading the remaining prerequisites of this tutorial then start [Task 1: Product Tour](https://training.h2o.ai/products/tutorial-1a-automatic-machine-learning-introduction-with-driverless-ai).
 
-  - Already have a **Two-Hour Test Drive** session? Continue reading the remaining prerequisites of this tutorial then start [Task 1: Product Tour](https://h2oai.github.io/tutorials/automatic-ml-intro-test-drive-tutorial/#2). 
+  - Already have a **Two-Hour Test Drive** session? Continue reading the remaining prerequisites of this tutorial then start [Task 1: Product Tour](https://training.h2o.ai/products/tutorial-1a-automatic-machine-learning-introduction-with-driverless-ai).
```
[the remark this answers is not in the document]
Comment (Castro38) [reply Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:37]: Great suggestion Sergio!

File: Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md (hunk 1 above)
Comment (Castro38) [reply Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:489]: Done

File: Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md (hunk 2 above)
Comment (Castro38) [reply Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:695]: Done

File: Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md (hunk 3 above)
Comment (Castro38) [reply Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:520]: Done

File: Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md (hunk 4 above)
Comment (Castro38) [reply Driverless AI/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md:743]: Updated the value to .4091 or about 41%. Thanks for catching it.

**Comment by Castro38** [comment]
> All

Although we can't point to a specific part of the tutorial we still need to point them to the tutorial itself. Rafael wants hyperlinks to point to the next tutorial for reference.

**Comment by 5675sp** [comment]
I think all hyperlinks specifying a particular task should stay as a hyperlink. Still, for the hyperlinks that are referencing a specific task in the tutorial at in view should be downgraded to regular text. With that, users will avoid opening unnecessary abs.

**Comment by Castro38** [comment]
> I think all hyperlinks specifying a particular task should stay as a hyperlink. Still, for the hyperlinks that are referencing a specific task in the tutorial at in view should be downgraded to regular text. With that, users will avoid opening unnecessary abs.

Let me bring it up to Rafael about what he wants to do in reference to all the tutorials that we have that point to specific sections of other tutorials since the goal was to have users be able to go back quickly with the link and review. For this release I will keep what we have and add this to the suggestions for all tutorials.

**Pull request merged**